packages/map/lib/src/leaflet/classes/leaflet-map.spec.ts

```diff
@@ -222,12 +222,12 @@ describe('The leaflet map', () => {
 		});
 
 		it('should find the current location', () => {
-			map.locate();
+			map.locate(16);
 			expect(locateSpy).toHaveBeenCalled();
 		});
 
 		it('should zoom to the current location', () => {
-			map.locate();
+			map.locate(16);
 			expect(setViewSpy).toHaveBeenCalledWith(options.center, 19);
```

Comment:
Is this working?
You're setting the zoom level at `16`, but are expecting zoom level `19`?